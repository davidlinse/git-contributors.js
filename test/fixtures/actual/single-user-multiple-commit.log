John Doe john@doe.com
 7 files changed, 7 insertions(+)
John Doe john@doe.com
 1 file changed, 3 insertions(+), 4 deletions(-)
John Doe john@doe.com
 1 file changed, 6  deletions(-)
