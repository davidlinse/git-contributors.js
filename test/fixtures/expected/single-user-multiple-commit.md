+ John Doe <john@doe.com>  (commits:3;insertions:10;deletions:10)
