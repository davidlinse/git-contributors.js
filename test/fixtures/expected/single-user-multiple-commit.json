[
  {
    "commits": 3,
    "name": "John Doe",
    "email": "john@doe.com",
    "percent": 100
  }
]
